README.md

```diff
@@ -23,11 +23,9 @@ FastFold provides a **high-performance implementation of Evoformer** with the fo
 
 ## Installation
 
-To install and use FastFold, you will need:
+To install FastFold, you will need:
 + Python 3.8 or 3.9.
-+ [NVIDIA CUDA](https://developer.nvidia.com/cuda-downloads) 11.1 or above
-+ PyTorch 1.10 or above 
-
++ [NVIDIA CUDA](https://developer.nvidia.com/cuda-downloads) 11.3 or above
```

Comment:
Why remove torch version?

Reply:
We will install torch via conda

Reply:
And we create a virtual env of python3.8 as well? I think these versions are for those who don't use conda or docker.

Reply:
Ok, I will set pytorch to be 1.12 or above. But it is difficult for us to set a virtual env because many packages (kalign2, hhsuite..) need to be installed from source if not use conda.

Reply:
Yes. In fact, conda users can install fastfold in their existing env rather than create a new one.